### PR TITLE
#162444011 CORS bug fix

### PR DESCRIPTION
**Pull request by Eyansky**
#### What does this PR do?
- Fix the issue with rejection requests.

#### Description of Task to be completed?
- Allow all origins to be accepted.

#### How should this be manually tested?
- Clone this repo
```
$ git clone https://github.com/andela/ah-orcas
```
- Navigate to this branch
```
$ git clone bg-cors-setup-162444011
```
- Install the requirements and run the migrations.
- Try accessing the endpoints from your react project.

#### Any background context you want to provide?
- This sorts to fix the issue on frontend that rejects the requests. The issue was CORS was not well implemented.
- I have tried this and it has worked locally. 

#### What are the relevant pivotal tracker stories?
[#162444011](https://www.pivotaltracker.com/story/show/162444011)

### CODE REVIEW

**Review by Gidraf — COMMENTED** [review]
Kindly change the PR format title

**Comment by code-sleuth** [comment]
Your commit message doesn't meet our agreed format. Your PR title too

**Review by code-sleuth — CHANGES_REQUESTED** [review]
cors not cor

**Comment by Eyansky** [comment]
@Gidraf @code-sleuth I have fixed the issues requested. Kindly check it again

File: requirements.txt [hunk 1]
```diff
@@ -67,4 +67,4 @@ whitenoise==4.1
 wrapt==1.10.11
 Django==2.1.2
 django-autoslug-iplweb==1.9.4
-django-versatileimagefield==1.10
+django-versatileimagefield==1.10
```
Comment (code-sleuth) [inline requirements.txt]: This is change is unnecessary considering there's no difference from the first dependency.

**Review by code-sleuth — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (Eyansky) [reply requirements.txt]: I have removed it. Kindly review it again.

**Review by code-sleuth — APPROVED** [review]
LGTM

**Review by Gidraf — APPROVED** [review]
LGTM

**Pull request merged**